qiskit_experiments/framework/restless_mixin.py

```diff
@@ -73,6 +79,8 @@ def enable_restless(
             override_processor_by_restless: If False, a data processor that is specified in the
                 analysis options of the experiment is not overridden by the restless data
                 processor. The default is True.
+            ignore_t1_check: If True, the default is False, then the ``rep_delay`` will not be
```

Comment:
I renamed it here [2f641b5](https://github.com/Qiskit/qiskit-experiments/pull/820/commits/2f641b5d06915cb86f46ab19a8e2b7ac5e993571) as the `suppress_t1_error` name is better. Regarding usability I think we are fine since

1. the default value is `False` and someone who wants to change this needs to explicitly write `supress_t1_error = True`
2. if the check fails then a warning is logged